src/test/kotlin/no/nav/pensjon/kalkulator/ufoere/api/UfoerepensjonControllerTest.kt

```diff
@@ -49,7 +49,7 @@ class UfoerepensjonControllerTest {
         `when`(ufoeretrygdService.harLoependeUfoerepensjon(DateFactory.date)).thenReturn(true)
 
         mvc.perform(
-            MockMvcRequestBuilders.post(URL)
+            MockMvcRequestBuilders.post(URL_UFOEREPENSJON)
```

Comment:
Er det riktig at det er URL_UFOEREPENSJON her, og ikke URL_HENT_UFOEREGRAD?

Reply:
Ja, fordi requesten skal hente løpende uførepensjon for bruker, mens uføregrad i nabotesten hentes fra en separat url som er bare for uføregrad. 🙂